v1/Makefile

```diff
@@ -11,7 +11,7 @@ DOCDIR := $(WORKDIR)/usr/share/doc/placemat
 EXAMPLEDIR := $(WORKDIR)/usr/share/doc/placemat/examples
 BASH_COMPLETION_DIR := $(WORKDIR)/etc/bash_completion.d
 BINDIR := $(WORKDIR)/usr/bin
-VERSION = 1.1.0-master
```

Comment:
I guess this is the version name for debian packages, not a git tag.

Reply:
Placemat V1 seems to have used a git tag for VERSION, so I used the same settings as here.
https://github.com/cybozu-go/placemat/blob/v1.5.3/.circleci/config.yml#L61